src/components/Card/Card.js

```diff
@@ -3,17 +3,20 @@ import PropTypes from 'prop-types';
 import './Card.css';
 import { Link } from 'react-router-dom';
 
-function Card({result}) {
+function Card({result, onDelete}) {
   const onImageError = (e) => {
     e.target.src = process.env.PUBLIC_URL + "/images/broken-record-lightcoral.png"
   }
   
   return (
-    <Link to={`/albums/${result.masterId}`} className='results--card' id={result.masterId}>
+  <div className='results--card' id={result.masterId}>
+    <Link to={`/albums/${result.masterId}`}>
       <img className='results--image' src={result.thumb ? result.thumb : process.env.PUBLIC_URL + "/images/broken-record-lightcoral.png"} alt={`${result.title} by ${result.artist}`} onError={onImageError}/>
       <p className='results--title'>{result.title.length > 20 ? result.title.slice(0, 20) + '...' : result.title}</p>
       <p className='results--artist'>{result.artist.length > 20 ? result.artist.slice(0, 20) + '...' : result.artist}</p>
     </Link>
+    <button className="delete-button" onClick={() => onDelete(result.masterId)}> Remove </button>
```

Comment:
I wonder if a better approach would be to have the button on album details view, the button could say add or remove conditionally

Reply:
I agree with this

Reply:
finished this ! with the insight from above